packages/collector/test/test_util/ProcessControls.js

```diff
@@ -190,7 +190,7 @@ class ProcessControls {
     return this.port;
   }
 
-  async start(retryTime) {
+  async start(retryTime, until) {
```

Comment:
NOTE: This is needed because we connect to the couchbase express app in the `before` function and it can take up to 30 seconds till everything is setup. Especially flushing the buckets can take some time.
Therefor we need to pass an `until` flag to ensure the test does not timeout early.

Reply:
I am fine with this either way, I think it's also fine to merge it as part of this PR, that is, I don't see a particular reason to extract it.